cmake/Modules/FindMKL.cmake

```diff
@@ -53,7 +53,12 @@ set(INTEL_OPT_ROOT "/opt/intel" CACHE PATH "Folder contains root-installed intel
   endif()
   cmake_dependent_option(MKL_USE_STATIC_LIBS "Use static libraries" ON "NOT MKL_USE_SINGLE_DYNAMIC_LIBRARY" OFF)
   cmake_dependent_option(MKL_MULTI_THREADED  "Use multi-threading"  ON "NOT MKL_USE_SINGLE_DYNAMIC_LIBRARY" OFF)
-  option(MKL_USE_ILP64 "Use ilp64 data model" OFF)
+  if(DEFINED USE_INT64_TENSOR_SIZE AND USE_INT64_TENSOR_SIZE)
+    message(STATUS "Enabling ILP64")
+    option(MKL_USE_ILP64 "Use ilp64 data model" ON)
```

Comment:
I think we should comment those out for now too. Andrzej includes this in his pr https://github.com/apache/incubator-mxnet/pull/19067/files#diff-192b252a8f0dda2d71d74cc9798f285340d597b8792a0cef1628d2fcaa13c26cR49

This is causing some build failures